storage/allocator_test.go

```diff
@@ -1158,55 +1158,55 @@ func Example_rebalancing() {
+	// 819 852 894 829 663 761 705 756 765 658 999 817 661 892 662 726 737 768 720 883
+	// 790 806 925 837 641 718 705 759 768 649 999 834 659 878 662 703 726 766 748 849
+	// 832 814 925 873 642 703 711 780 771 673 999 868 711 894 677 715 736 767 744 885
+	// 849 857 957 912 669 745 746 836 792 692 999 918 772 905 723 776 745 821 734 892
+	// 834 833 992 930 704 740 774 848 829 727 999 912 788 906 721 797 770 861 754 878
+	// 840 820 983 936 727 754 794 855 866 755 999 909 795 936 695 796 776 858 760 882
+	// 812 804 999 922 768 791 829 825 894 783 982 896 793 936 718 816 760 868 791 926
+	// 847 847 983 914 762 822 856 850 880 819 999 872 877 933 752 844 842 874 851 873
+	// 840 874 966 904 777 879 858 885 860 860 999 886 851 916 752 851 853 878 854 887
+	// 858 868 955 882 801 893 857 892 874 856 999 885 839 914 753 843 871 874 857 890
+	// 874 869 931 925 847 900 883 885 872 907 999 894 832 937 829 870 908 901 891 908
+	// 908 924 921 901 900 932 899 924 881 932 999 923 867 938 893 892 930 902 942 937
+	// 904 945 920 898 912 938 927 935 894 934 999 908 894 943 900 929 958 915 925 954
+	// 936 991 952 890 953 979 952 963 942 968 996 952 923 969 937 958 999 964 932 985
+	// 937 989 983 900 944 991 979 974 959 946 984 965 931 979 914 965 999 972 938 972
+	// 951 996 973 911 953 999 951 965 942 957 964 972 923 957 923 960 975 966 933 957
+	// 966 999 988 932 960 980 967 956 937 963 980 972 934 970 957 963 971 988 949 971
+	// 951 961 999 926 955 960 966 962 930 966 985 975 960 960 975 952 985 996 934 993
+	// 951 961 991 943 961 959 980 950 917 968 999 957 975 969 970 952 975 983 931 991
+	// 918 948 999 932 932 959 974 943 939 951 975 949 965 973 962 933 961 972 921 978
+	// 957 953 999 916 947 961 970 956 926 966 967 950 969 985 973 925 970 988 926 988
+	// 951 925 999 888 917 944 974 944 918 941 946 960 959 988 968 911 969 951 941 967
+	// 939 933 999 914 912 953 963 939 925 922 959 959 951 959 968 914 968 960 945 950
+	// 951 935 999 932 934 953 970 929 942 938 948 956 965 968 958 933 980 964 955 960
+	// 935 930 999 936 928 925 954 918 927 939 942 946 948 964 960 922 969 950 954 944
+	// 938 950 999 961 938 921 961 915 946 963 958 967 943 976 971 946 970 966 947 952
+	// 938 965 999 987 922 915 963 921 959 970 973 950 942 994 960 950 992 968 940 951
+	// 928 957 959 957 916 905 969 921 955 981 969 927 939 963 940 948 999 962 951 937
+	// 938 955 945 951 922 892 972 905 952 970 964 945 947 959 945 942 999 967 959 946
+	// 935 954 928 948 923 890 965 893 942 971 958 936 952 949 950 940 999 965 943 935
+	// 951 974 953 971 931 915 991 954 964 999 981 961 983 978 971 966 995 984 981 971
+	// 962 974 946 976 924 912 999 949 958 997 965 964 990 976 965 962 993 965 974 959
+	// 970 962 953 964 929 915 975 949 955 999 962 963 982 985 948 968 974 965 972 934
+	// 974 964 935 947 923 902 964 938 957 999 963 968 968 979 958 965 963 952 967 937
+	// 957 971 936 947 933 905 962 932 951 999 949 968 966 978 962 970 951 944 963 934
```

Comment:
What should we be looking for in this test? The the output often changes completely and we seem to just rubber-stamp whatever the new output becomes. Is there a better way to do this test so the output isn't so volatile?